web/l10n/de.json

```diff
@@ -144,6 +144,8 @@
     "commandData": "Daten",
     "commandPhone": "Rufnummer",
     "commandMessage": "Nachricht",
+    "commandTimezoneOffset": "Zeitverschiebung zu UTC",
```

Comment:
You shouldn't change any of the localization files directly. All localization should be done via Transifex website.